lighttpd_rust_amalgamated/Cargo.toml

```diff
@@ -17,5 +17,7 @@ crate-type = ["staticlib", "rlib"]
 path = "src/lighttpd_amalgamated.rs"
 name = "lighttpd_amalgamated"
 [dependencies]
-c2rust-bitfields= "0.3"
+c2rust-analysis-rt = { version = "0.17.0", path = "../../c2rust/analysis/runtime", optional = true }
+c2rust-bitfields = { path = "../../c2rust/c2rust-bitfields"}
+c2rust-bitfields-derive = { path = "../../c2rust/c2rust-bitfields-derive"}
```

Comment:
Why are the relative paths necessary?  That'll hardcode the relative location of `lighttpd-rust` and we don't want that.

Reply:
addressed in [7cb2e3d](https://github.com/immunant/lighttpd-rust/pull/6/commits/7cb2e3d190a375b49177aea52365cab004d7a139)